articles/115_topic_and_service_name_mapping.md

```diff
@@ -119,10 +120,10 @@ Topic and service names:
 
 - must not end with a forward slash (`/`)
 
-The last token is the topic or service basename, and any preceding tokens comprise the namespace of the topic or service.
+The last token is the topic or service basename, and any preceding tokens make up the namespace of the topic or service.
 
-For example, the topic name `/foo/bar/baz` comprises of a topic or service with the basename `baz` in the `/foo/bar` namespace.
-In another example, the name `/foo` splits into one token, such that it comprises of a topic or service with the basename `foo` in the `/` namespace (the root namespace).
+For example, the topic name `/foo/bar/baz` comprises a topic or service with the basename `baz` in the `/foo/bar` namespace.
+In another example, the name `/foo` splits into one token, such that it comprises a topic or service with the basename `foo` in the `/` namespace (the root namespace).
```

Comment:
I don't use comprise often so it always sounds weird to me. "is composed of" is what we want I believe